python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -924,6 +1095,18 @@ def get_startup_program(self,
         Returns:
             Program: parameter server side startup program.
         """
+        sys.stderr.write("get_startup_program() is deprecated, call \
+get_pserver_programs() to get pserver main and startup \
+in a single call.")
+        if pserver_program != None:
+            sys.stderr.write("passing pserver_program to get_startup_program() \
+is deprecated, you can use new API get_pserver_programs() to \
+get both pserver main program and startup program.")
+        if startup_program != None:
+            sys.stderr.write("passing startup_program to get_startup_program() \
```

Comment:
Should remove this for now, need a better way to tell the deprecation

Reply:
done